connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -72,6 +72,10 @@
                     + " not per task. All task have shutdown triggered, then they are waited on sequentially.";
     private static final String TASK_SHUTDOWN_GRACEFUL_TIMEOUT_MS_DEFAULT = "5000";
 
+    public final static String OFFSET_STORAGE_CLASS_CONFIG = "offset.storage.class";
```

Comment:
I'm hesitant to add a config like this. Right now there is only one implementation of `OffsetBackingStore` that makes sense for each of the ways we can execute Connect -- standalone mode uses `FileOffsetBackingStore` and distributed mode uses `KafkaOffsetBackingStore`. I suppose in the case of standalone mode it is possible to use `KafkaOffsetBackingStore` instead?

What's the motivation for adding this configuration option in this patch? It looks like you added the other configs (offset storage topic, offset storage file) in the way I expected, in the `StandaloneConfig`/`DistributedConfig`, so I'm not sure that this config is needed. Also, it doesn't seem to have been added to the `ConfigDef` below?